### ISSUE TITLE
Fix addons in Storybook

### ISSUE THREAD

**Issue by blaketarter** [cremalab/crema-app-native#13]
Storybook addons aren't working in the current setup 😬 .

We need to:
- [ ] rename `./storybook/addons.ts` -> `./storybook/rn-addons.ts`
- [ ] change `@storybook/addon-actions` to `@storybook/addon-ondevice-actions`
- [ ] import `./storybook/rn-addons.ts` into `./storybook/index.ts`
- [ ] test it 🤦‍♂

**Comment by luiz-chagas** [comment]
Storybook for RN is in a bad place right now, they even removed their react native example from the `examples` folder.

Lots of issues here, still can't get addons to work. https://github.com/storybookjs/react-native/issues

**Comment by blaketarter** [comment]
Yeah it looks like they are in the middle of breaking all of the react-native code into a separate repository, so until that is finished I can't see addons being fixed..
